tiledb/sm/subarray/subarray.cc

```diff
@@ -1133,7 +1126,7 @@ const std::vector<std::vector<uint8_t>>& Subarray::tile_coords() const {
 }
 
 const std::vector<std::vector<TileOverlap>>& Subarray::tile_overlap() const {
-  return tile_overlap_;
+  return *tile_overlap_;
```

Comment:
What if the pointer is `nullptr`? Do we need an `assert` here just in case?

Reply:
Good point. I assume that this is not used until the caller knows that the tile overlap has been computed. I'll add an assert.